run/README.namelist

```diff
@@ -334,11 +334,11 @@ lateral BC computations.
  interp_method_type                  = 1 ! bi-linear interpolation
                                      = 2 ! SINT, (default)
                                      = 3 ! nearest neighbor - only to be used for 
-                                         ! testing purposes
+                                           testing purposes
```

Comment:
@smileMchen 
Ming,
Here and a few line below, it looks like the original file had the correct "!", and now it has been removed.